agents/src/index.ts

```diff
@@ -15,6 +15,19 @@ import * as multimodal from './multimodal/index.js';
 import * as stt from './stt/index.js';
 import * as tts from './tts/index.js';
 
+const isCommonJS = (): boolean => {
+  try {
+    return !!require;
+  } catch {
+    return false;
+  }
+};
+if (isCommonJS()) {
+  throw new ReferenceError(
+    '@livekit/agents cannot be used in a CommonJS environment. Please set your project to use ES modules.',
```

Comment:
we could explicitly tell them to set the module field i. package.json, that might be clearer